ios-target/Extensions/UIColorExtension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  UIColorExtension.swift
+//  ios-target
+//
+//  Created by sol manini on 6/25/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import UIKit
+
+extension UIColor {
+  
+  static let brownGrey = UIColor(white: 166.0 / 255.0, alpha: 1.0)
+  static let macaroniAndCheese70 = UIColor(red: 239.0 / 255.0, green: 197.0 / 255.0, blue: 55.0 / 255.0, alpha: 0.7)
+  static let black = UIColor(red: 223, green: 222, blue: 223)
```

Comment:
redefining the wheel are ya? 
I recommend this web page to name colors:
http://chir.ag/projects/name-that-color/#6195ED

Reply:
hahaha design team used those names 🤷🏽‍♀️